### PR TITLE
Fix inconsistent resolution order with zope.interface v5.

### PR DESCRIPTION
**Pull request by janjaapdriessen**
I fixed inconsistent [IRO in ZODB](https://github.com/zopefoundation/ZODB/pull/310/files), so travis won't like this PR until we have a release of ZODB.

### CODE REVIEW

File: src/zope/session/http.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ class ICookieClientIdManager(IClientIdManager):
         )
 
 
-@implementer(IClientIdManager, ICookieClientIdManager)
+@implementer(ICookieClientIdManager)
```
Comment (jamadden) [inline src/zope/session/http.py:133]: `class ICookieClientIdManager(IClientIdManager):` is right above this, so this LGTM.

**Review by jamadden — COMMENTED** [review]
LGTM, assuming the tests go green once ZODB is updated.

**Comment by icemac** [comment]
As now tests in Travis are green: Is there still something to do before this PR can be merged?

**Comment by janjaapdriessen** [comment]
Yes @icemac please merge this PR.

**Comment by icemac** [comment]
Thank you!

**Comment by janjaapdriessen** [comment]
@icemac would you make a pypi release please? I don't have pypi rights.

**Comment by mgedmin** [comment]
> I don't have pypi rights.

Would you like to have them?  What's your PyPI username?

File: CHANGES.rst [hunk 2]
```diff
@@ -40,6 +40,7 @@
 
 - Add support for testing on Travis.
 
+- Fix inconsistent resolution order with zope.interface v5.
```
Comment (mgedmin) [inline CHANGES.rst:43]: This is, uh, in the wrong spot.  I'll fix it in master.

**Comment by mgedmin** [comment]
> would you make a pypi release please?

4.4.0 is out.

**Comment by janjaapdriessen** [comment]
> > I don't have pypi rights.
> 
> Would you like to have them? What's your PyPI username?

Yes please, I'd like to have pypi rights - I am `janjaapdriessen` on pypi.

**Comment by mgedmin** [comment]
> Yes please, I'd like to have pypi rights - I am janjaapdriessen on pypi.

You now have them.

**Pull request merged**